actions/workflows/promote_package.yaml

```diff
@@ -0,0 +1,100 @@
+---
+version: '2.0'
+
+st2cd.promote_package:
+    description: Promote a package to production.
+    type: direct
+    input:
+      - package
+      - version
+      - revision
+      - os
+      - repo
+    tasks:
+
+      init:
+        action: core.local
+        input:
+          cmd: "mkdir -p /tmp/st2-up/{ubuntu/trusty,debian/wheezy,debian/jessie,el/6,el/7}"
+        publish:
+          rpm: "<% $.package %>-<% $.version %>-<% $.revision %>.x86_64.rpm"
```

Comment:
@emedvedev Actually for now real case is @dennybaa case. If you convince him not to use `_all.deb` (I tried once :) - everything will work with such hardcoding way.


If not - you need to fetch PackageCloud API to get real file name depending on pg_name (ex: `st2`), revision, os, repo.
See example API request: https://github.com/StackStorm/st2-packages/blob/master/.circle/packagecloud.sh#L211